package.json

```diff
@@ -35,5 +35,8 @@
     "ts-jest": "^27.1.0",
     "typescript": "^4.2.4"
   },
+  "resolutions": {
+    "fs-extra": "^9.0.0"
```

Comment:
Related to `fs.realpath.native` being `undefined`, causing import of `fs-extra@10.0.0` to throw error.

Defensively patched in https://github.com/jprichardson/node-fs-extra/pull/953 and released in [`fs-extra@10.1.0`](https://github.com/jprichardson/node-fs-extra/blob/10.1.0/CHANGELOG.md).

Might want to bump it up instead, maybe?

But still, why is `fs.realpath.native` `undefined`?  That is a root cause to be investigated.

Reply:
@peteriman Thanks a lot for putting a fix in for this issue **and** for following up here!

It's not clear to me why `fs.realpath.native` is `undefined` either. I may spend a little time trying to figure that out and get back to you if I find anything.

Thanks again!

Reply:
While `fs-extra` no longer throws error on require/import, if someone still uses `fs. realpath.native` (be it from `fs-extra` or from native `fs`) they would still face errors. 

@austinkelleher You might want to `console.log(typeof fs.realpath.native)` at the first line of code as a pre-requisite, then the subsequently before and after every require/import to identify the offending dependency that "deleted" it.